resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,258 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+import "./device-verification"
+
+ApplicationWindow{
```

Comment:
Why ApplicationWindow and not just Window?

Reply:
Using window says palette is a non-existent property, that's why